python/dgl/partition.py

```diff
@@ -592,5 +592,21 @@ def map_to_global(self, idxs, part_id):
             )
         )
 
+    def generate_permutation(self, idxs):
```

Comment:
Add unittest for this method?

Reply:
Added.
PS: `GeneratePermutation` has its C++ test.